source/administration/mobile-changelog.md

```diff
@@ -1,5 +1,37 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.42.0 Release
+- Release Date: April 16, 2021
+- Server Versions Supported: Server v5.25+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.31 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.25 has ended and upgrading to server ESR v5.31 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Improvements
+ - Apps Framework
```

Comment:
Needs a better description, as well as any notes on compatibility / breaking changes (if any). @larkox

Reply:
Shouldn't we be asking better description to the PM instead?

Also this should be highlighted somehow, I do believe this is massive!

Reply:
@aaronrothschild would be the right person to give a good description here.

Regarding compatibility and breaking changes, it should include none.

Reply:
Also, compatibility wise, it depends on a feature flag added on on this PR: https://github.com/mattermost/mattermost-server/pull/16851
According to the milestone on the pr, it is server version 5.35.